examples/colab/text_classification_with_tf_hub_on_kaggle.ipynb

```diff
@@ -170,7 +170,7 @@
         "      raise no_file\n",
         "    \n",
```

Comment:
One thing that happened to me when running this colab. If one calls get_kaggle_credentials() and uploads the wrong file, they can never recover with this through the colab.

Could you please change the function to do something like:

def get_kaggle_credentials(force_upload=False)

Then inside the function if force_upload==True, delete the token_file if it existed and upload a new one.

Reply:
I changed the get_kaggle_credential function to do force upload.